packages/manager/apps/web/client/app/private-database/cpu-throttle/private-database-cpu-throttle.html

```diff
@@ -0,0 +1,54 @@
+<div data-ng-controller="PrivateDatabaseCpuThrottleCtrl as cpuThrottleCtrl">
+    <div
```

Comment:
current practice is to use a specific `modal` layout as wizard is deprecated.
Also are we sure about datagrid display in a modal ?

Reply:
As with the controller, I am doing this to keep the codebase consistent.